### DISCUSSION TITLE
我的时间管理工具 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#150] [Announcements]
# 我的时间管理工具 · BMPI

如何进行时间管理？本文介绍了我近3年对时间管理工具（GTD）的思考，最终构建了一套以VSCode工具为基础结合邮件通知的可以长期免费使用的时间任务目标管理工具，帮助你成为一个高效率的人

https://www.bmpi.dev/self/gtd-tools-i-used/

**Comment by Vant1032** [comment]
为何不使用滴答清单，全平台同步且完美支持上述功能
